roles/django_app_docker/templates/networks.yml.j2

```diff
@@ -3,5 +3,6 @@ binds containers ONLY to the app network. -#}
 networks:
   - name: "{{ django_app_docker_name_prefix }}"
 network_mode: "{{ django_app_docker_name_prefix }}"
-purge_networks: true
 networks_cli_compatible: true
+{% if ansible_version.major == 2 and ansible_version.minor <= 10 %}purge_networks: true{% endif %}
```

Comment:
I hesitate 🤔 I had locally 2.17.14 for the ansible-core version in commonground-deployment, so then I would be missing this.

Since we don't specify in all repo specific ansible version requirements, maybe we should make all the upgrades for newer ansible versions in one go and then update all the repositories 🤔

Reply:
isn't this the idea, that you would only have purge_networks for older ansible-core versions?

ansible-core 2.10 is more or less ancient ( https://endoflife.date/ansible-core ) however I noticed that the Sjoerd-repo's still explicitly state installing ansible==10.x